### DISCUSSION TITLE
Is this plugin dead?

### DISCUSSION THREAD

**Discussion by mklenbw** [UnitedIncome/serverless-python-requirements#816] [General]
Hello team, nobody seems to fix all the errors since the original maintainer left. Is there a roadmap for this plugin or will it be integrated with serverless-v4?

Contributions on serverless are going down as well. How will this end up?

**Comment by pgrzesik** [comment]
Hey @mklenbw 👋 The plugin is not necessarily dead as I'm trying to take a look at issues/PRs from time to time, but I have to admit that my time is quite limited lately, sorry about that. 

As for Serverless V4, that should be probably asked on the main serverless repo.
